gnovm/stdlibs/std/banker.go

```diff
@@ -88,10 +92,30 @@ func X_bankerTotalCoin(m *gno.Machine, bt uint8, denom string) int64 {
 
 func X_bankerIssueCoin(m *gno.Machine, bt uint8, addr string, denom string, amount int64) {
 	// gno checks for bt == RealmIssue
-	m.Context.(ExecContext).Banker.IssueCoin(crypto.Bech32Address(addr), denom, amount)
+
+	// check origin denom format
+	matched, err := regexp.MatchString(denomRegex, denom)
```

Comment:
This slipped my eye in an earlier review

Please don't use `regexp.Match` or `MatchString`; they will have to re-compile the regex at each execution. They can be stored as singletons (global variables)